bzt/modules/python.py

```diff
@@ -334,6 +342,18 @@ def add_imports(self):
             imports.text = self.IMPORTS_SELENIUM
         return imports
 
+    def gen_global_vars(self):
+        variables = self.scenario.get("variables")
+        stmts = [
+            "_vars = {}",
+            "_tpl = selenium_taurus_extras.Template(_vars)"
+        ]
+
+        for key in sorted(variables.keys()):
+            stmts.append("_vars['%s'] = %r" % (key, variables[key]))
+        stmts.append("")
+        return self.gen_statement("\n".join(stmts), indent=0)
```

Comment:
Technically, it would be nicer to return a list of statements instead of a single `\n`-joined line.

Reply:
Also, i don't think it should generate variable initialization lines if there were no variables declared.

Reply:
Yep, sorry for the list of statements, I already solved that mistake. 3c281c79abf22668e403b8473bc625193e7fcbae

Related to the non-generation of _vars or _tmpl, will come commands that in runtime will create and use variables, the structure must be created before (as JMeter).
I find the difficulty of having to preprocess all the actions previously to be able to know if any of them within a string handle or not a template of variables.

Reply:
Ok, having untime variable creation in mind, i agree with always generating `_vars` and `_tmpl`.